src/ui/Controls/SETextBox.cs

```diff
@@ -13,21 +13,6 @@ namespace Nikse.SubtitleEdit.Controls
     /// </summary>
     public sealed class SETextBox : Panel, ISelectedText
     {
-        // ReSharper disable once InconsistentNaming
-        public new event EventHandler TextChanged;
-        // ReSharper disable once InconsistentNaming
-        public new event KeyEventHandler KeyDown;
-        // ReSharper disable once InconsistentNaming
-        public new event MouseEventHandler MouseClick;
-        // ReSharper disable once InconsistentNaming
-        public new event EventHandler Enter;
-        // ReSharper disable once InconsistentNaming
-        public new event KeyEventHandler KeyUp;
-        // ReSharper disable once InconsistentNaming
-        public new event EventHandler Leave;
-        // ReSharper disable once InconsistentNaming
-        public new event MouseEventHandler MouseMove;
```

Comment:
Any other control that want to hook to these event can do it via the parent class 
and if we want to invoke these event we can just use for example "OnEnter(EventArgs)" so no need for this!